services/ymax-planner/README.md

```diff
@@ -73,6 +73,7 @@ yarn test
 
 Environment variables:
 
+- `CONTRACT_INSTANCE`: Contract instance identifier, either "ymax0" (dev) or "ymax1" (prod) (default "ymax0")
```

Comment:
This seems like it should be required; anything else will likely lead to confusion about why the planner is interfacing with an old contract.

Reply:
Done in #12063.